src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+signupBanner:
+  text: Monitor and improve your entire stack. 100GB free. Forever.
+freshnessValidatedDate: 2024-03-dd
+---
+
+import NRAlertsMain from 'images/alerts_screenshot-crop_main.webp'
+
+import NRApmMain from 'images/apm_screenshot-crop_main.webp'
+
+import NRBrowserMain from 'images/browser_screenshot-crop_main.webp'
+
+import NRDashboardsMain from 'images/dashboards_screenshot-crop_main.webp'
+
+import NRErrosInboxMain from 'images/errors-inbox_screenshot-crop_main.webp'
+
+import NRIastMain from 'images/iast_screenshot-crop_main.webp'
+
+import NRInfraMain from 'images/infrastructure_screenshot-crop_main.webp'
+
+import NRLambdaMain from 'images/lambda_screenshot-crop_main.webp'
+
+import NRLiveArchivesMain from 'images/live-archives_screenshot-crop_main.webp'
+
+import NRLogsMain from 'images/logs_screenshot-crop_main.webp'
+
+import NRK8sMain from 'images/kubernetes_screenshot-crop_main.webp'
+
+import NRMobileMain from 'images/mobile_screenshot-crop_main.webp'
+
+import NRNetworkMain from 'images/network_screenshot-crop_main.webp'
+
+import NROpenTelemetryMain from 'images/opentelemetry_screenshot-crop_main.webp'
+
+import NRServiceLevelsMain from 'images/slm_screenshot-crop_main.webp'
+
+import NRSyntheticMain from 'images/synthetic_screenshot-crop_main.webp'
+
+import NRVulnManagementMain from 'images/vulnerability-management_screenshot-crop_main.webp'
+
+
+Welcome to the New Relic observability platform! Our platform gives you deep visibility into your systems. It helps you solve problems you may not have been aware of, regardless of the technology you use. See how New Relic works [here](#how-it-works) or if you want to start now, let's [get started](#get-started).
+
+New Relic provides many capabilities to improve visibility. They let you monitor, troubleshoot, and improve digital performance. Here's a brief overview of our most popular capabilities and features.
+
+
+# Alerts [#alerts]
+<SideBySide>
+  <Side>
+Alerts help you identify issues that look unusual in your applications and code. New Relic includes some <InlinePopover type="alerts" /> when you install it and you can also create your alerts. You receive alerts through tools like PagerDuty, ServiceNow, Jira, Slack, and more. Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue. 
+  
+Learn more about alerts [here](/docs/alerts-applied-intelligence/overview/).
+
+  </Side>
+  <Side>
+    <img
+      title="New Relic alerts"
+      alt="New Relic alerts"
+      src={NRAlertsMain}
+    />
+  </Side>
+  </SideBySide>
+
+# APM [#apm]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic APM"
+    alt="New Relic APM"
+    src={NRApmMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic instruments your application by using one of our many language agents. These agents collect metrics from your application. They send the metrics to <InlinePopover type="apm" />. It lets you monitor it using pre-built dashboards. 
```

Comment:
"It lets you monitor it", I'd clarify this a bit more. What lets you monitor what, exactly?